<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbPortName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdN1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 1</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdN2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLed1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLed2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbADC2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbADC1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSerialStatus_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>85</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected to </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbJsonSent">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>85</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>JSON sent: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbJsonReceived">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>86</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>JSON received</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSent">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>85</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbReceived">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>85</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
